PLAYER/SetPlayerInvincible.md

```diff
@@ -5,16 +5,12 @@ ns: PLAYER
 
 ```c
 // 0x239528EACDC3E7DE 0xDFB9A2A2
-void SET_PLAYER_INVINCIBLE(Player player, BOOL toggle);
+void SET_PLAYER_INVINCIBLE(Player player, BOOL invicible);
```

Comment:
I think `toggle` was fine, additionally if you want to rename to `invincible` you might be better off using something such as `bInvincible` to state in writing that it's a boolean (useful for dynamically typed languages, especially if autocomplete is used).